From 6a41846f3191afe48144481bbfa2fd4edafdc8d5 Mon Sep 17 00:00:00 2001
From: Sanskriti Sharma <sansharm@redhat.com>
Date: Wed, 21 Nov 2018 10:28:44 -0500
Subject: [PATCH] Added memhog.8 to Makefile.am Signed-off-by: Sanskriti Sharma
 <sansharm@redhat.com>

---
 Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile.am b/Makefile.am
index 1c4266d..ab61d42 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -12,7 +12,7 @@ include_HEADERS = numa.h numacompat1.h numaif.h
 
 noinst_HEADERS = numaint.h util.h
 
-dist_man_MANS = move_pages.2 numa.3 numactl.8 numastat.8 migratepages.8 migspeed.8
+dist_man_MANS = move_pages.2 numa.3 numactl.8 numastat.8 migratepages.8 migspeed.8 memhog.8
 
 EXTRA_DIST = README.md INSTALL.md
 
-- 
2.7.5

